src/components/Layout/Chat/Chat.jsx

```diff
@@ -62,8 +62,8 @@ const ChatHeader = styled.div`
 
 const ChatBody = styled.div`
   position: relative;
-  height: 100%;
-  overflow-y: auto;
+  height: 250px;
+  overflow-y: scroll;
```

Comment:
Why was this changed? :thinking: 
That wasnt your job...... :joy: 

Unless you can justify this CSS change I wont accept it. :)

Reply:
I was playing around with the autoscroll feature on Friday 🤦🏾‍♂️.
Removing now